packages/core/src/extractors/prompts.ts

```diff
@@ -0,0 +1,52 @@
+export const defaultKeywordExtractorPromptTemplate = ({
+  context_str,
+  keywords,
+}: {
+  context_str: string;
+  keywords: number;
+}) => `
+  /${context_str}. Give ${keywords} unique keywords for this \
```

Comment:
Unlike Python, the backslash doesn't work for JS template literals to continue the line. I would just write out the whole line, but if you really want you can use string concatenation.

Similarly the slash (forward slash) at the beginning of the template literal doesn't do want you want it to, which I'm guessing is to not have the newline.